standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -3783,9 +3785,10 @@ public boolean submitForCleanup(CompactionRequest rqst, long highestWriteId, lon
           buf.append("\"CQ_PARTITION\", ");
           params.add(partName);
         }
-        buf.append("\"CQ_STATE\", \"CQ_TYPE\"");
+        buf.append("\"CQ_STATE\", \"CQ_TYPE\", \"CQ_POOL_NAME\"");
```

Comment:
is there special pool-based handling in Cleaner? if not, why do we need to propagate pool here?

Reply:
You are right, this method creates the entries with `READY_FOR_CLEANING` status. If there's no use case where these requests are somehow picked up by the Worker, this change can be reverted.